jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -0,0 +1,1217 @@
+* Create a Nomad Autoscaler IAM user manually with the <<iam-policy-for-nomad-autoscaler,IAM policy below>>. Then, generate an access and secret key for this user.
+* You may create a https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[Role for Service Accounts] for Nomad Autoscaler and attach the <<iam-policy-for-nomad-autoscaler,IAM policy below>>:
+
+When using access keys and secrets, you have two options for configuration:
+
+[.tab.awsautoscaler.CircleCI_creates_secret]
+--
+**Option 1:** CircleCI creates the secret for you.
+
+Add the access key and secret to your values.yaml file with any additional configuration required.
+
+[source, yaml]
+----
+nomad:
+...
+  auto_scaler:
+    aws:
+      accessKey: "<access-key>"
+      secretKey: "<secret-key>"
+----
+--
+
+[.tab.awsautoscaler.You_create_secret]
+--
+**Option 2:** Create the secret yourself
+
+Instead of storing the access key and secret in your values file, you may create the secret yourself. Please note, an additional field is required for this secret, as outlined below.
+
+First, add your access key, secret key, and region to the following text, and encode it all with base64.
+
+[source,shell]
+----
+ADDITIONAL_CONFIG=`cat << EOF | base64
+target "aws-asg" {
+  driver = "aws-asg"
+  config = {
+    aws_region = "<aws-region>"
+    aws_access_key_id = "<access-key>"
+    aws_secret_access_key = "<secret-key>"
+  }
+}
+EOF`
+----
+
+Then, using that additional base64 encoded config, create the secret.
+
+[source, shell]
+----
+# With the base64-encoded additional config from above
+kubectl create secret generic nomad-autoscaler-secret \
+  --from-literal=secret.hcl=$ADDITIONAL_CONFIG
+----
+--
+
+
+[#iam-policy-for-nomad-autoscaler]
+===== IAM policy for Nomad Autoscaler
+
+[source, json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:CreateOrUpdateTags",
+                "autoscaling:UpdateAutoScalingGroup",
+                "autoscaling:TerminateInstanceInAutoScalingGroup"
+            ],
+            "Resource": "<<Your Autoscaling Group ARN>>"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeScalingActivities",
+                "autoscaling:DescribeAutoScalingGroups"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+
+endif::env-gcp[]
+
+ifndef::env-aws[]
+
+[#gcp-service-account]
+==== GCP autoscaler service account
```

Comment:
@circleci/server could someone check this section for accuracy?

Reply:
This looks right to me, just had the one comment